Add --group option

hongkongkiwi opened the pull request:
Added a group option so we can pass different group when selecting a server.

You can list groups with `nordvpn groups`

hongkongkiwi committed:
Add --group option

sfiorini commented:
Thanks @hongkongkiwi . I will review and test over the weekend then merge. Sorry for being a bit absent. Extremely busy with work. :) Appreciate your help

hongkongkiwi committed:
Added option to specify output conf file 

Added option to specify output file and additionally made sure all files are written to tmp directory

hongkongkiwi committed:
Update NordVpnToWireguard.sh

Updated version and fixed some directory issues

hongkongkiwi committed:
Update NordVpnToWireguard.sh

Whitelisted port 22 for SSH so we don't block ssh incase the VPN is stuck on

hongkongkiwi committed:
Update NordVpnToWireguard.sh

Set technology to NordLynx and then back to original technology afterwards

sfiorini commented:
Version 0.2.0 includes the option to create a configuration for a servers group.
